packages/fuels-core/src/types/wrappers/transaction.rs

```diff
@@ -248,6 +260,8 @@ pub trait Transaction:
 
     fn with_maturity(self, maturity: u32) -> Self;
 
+    fn expiration(&self) -> Option<u64>;
```

Comment:
should we a setter `with_expiration` here, like we have for maturity

Reply:
Good question. Well other policies are also missing the `with_` methods. We can either remove it or add them for all? What do you think?

Reply:
`with_maturity` changes the `tx_id` so I would like to remove it.